src/main/java/net/sf/jabref/MetaData.java

```diff
@@ -30,9 +30,10 @@
 
     public static final String META_FLAG = "jabref-meta: ";
     public static final String SAVE_ORDER_CONFIG = "saveOrderConfig";
+    public static final String SAVE_ACTIONS = "saveActions";
```

Comment:
The save actions are not widely used yet (no UI in a official release), so we might get away with changing the meta key. If we unify the cleanup and save actions, maybe something like "cleanupActions" would be better?